templates/intro.tmpl.html

```diff
@@ -38,17 +38,17 @@
         <div class="container content-width">
             <main class="frontpage">
                 <div class="thumbnail">
-                    <a href="http://tanx.playcanvas.com"><img src="/images/thumbnails/Tanx_Thumb.png" /></a>
+                    <a href="https://tanx.io"><img src="/images/thumbnails/Tanx_Thumb.png" /></a>
                     <h1>{{#i18n}}Games{{/i18n}}</h1>
                     <p>{{#i18n}}Awesome games{{/i18n}}</p>
                 </div>
                 <div class="thumbnail">
-                    <a href="http://car.playcanvas.com"><img src="/images/thumbnails/BMW_Thumb.png" /></a>
+                    <a href="https://car.playcanvas.com"><img src="/images/thumbnails/BMW_Thumb.png" /></a>
```

Comment:
I don't think `https://car.playcanvas.com` is valid. I'd suggest just using the project URL: https://playcanv.as/p/RqJJ9oU9/

Reply:
Oh that's interesting, http://car.playcanvas.com is fine but not https. 

I change to use the .as link instead regardless as it's safer.